### DISCUSSION TITLE
How to not have separate sitemap files (sitemap.xml and sitemap-0.xml) and have just 1

### DISCUSSION THREAD

**Discussion by vinaynair39** [iamvishnusankar/next-sitemap#333] [Q&A] [answered]
First of all thanks for creating this wonderful library!

When I upgraded to the latest version, I see that we're now generating 2 sitemaps,
   ○sitemap.xml (index)
   ○ sitemap-0.xml
   
Is there any way to not let this happen? I mean I just want 1 sitemap, just like how it was before.

**Comment by iamvishnusankar** [comment, marked as answer]
@vinaynair39 Please read: https://github.com/iamvishnusankar/next-sitemap#index-sitemaps
